Marlin/Marlin_main.cpp

```diff
@@ -8276,8 +8271,8 @@ void ok_to_send() {
       SERIAL_ECHOPAIR(" L=", L);
       SERIAL_ECHOPAIR(" R=", R);
       SERIAL_ECHOLNPAIR(" offset=", offset);
+      last_offset = offset;
```

Comment:
The problem is in the fact, that the function is called once for every segment of delta movement. Not only for destination of the movement. So leaving the assignment outside the block, the report is printed only ones at begining. In normal situation there cannot be difference greater than 0.1mm as the function is never called for coordinates far than about 0.5mm from the last call.

The report should be modified in some way, however hopefully this is only for debug.